### PR TITLE
Remove html comparisons from AdminSearchPageUiTest #4347

### LINKED ISSUE #4347: Remove html comparisons from AdminSearchPageUiTest

### PR DESCRIPTION
**Pull request by taniach**
Fixes #4347 

@JosephineKwa Can you please review this? I tested it on both the staging server and the dev server.

### CODE REVIEW

**Comment by JosephineKwa** [comment]
Remember to remove the unused html pages :)

File: src/test/java/teammates/test/cases/ui/browsertests/AdminSearchPageUiTest.java [hunk 1]
```diff
@@ -68,6 +78,94 @@ private AdminSearchPage getAdminSearchPage(String instructorId) {
         return loginAdminToPage(browser, commentsPageUrl, AdminSearchPage.class);
     }
     
+    private boolean isPageTitleCorrect() {
+        return searchPage.getPageTitle().equals("Admin Search");
+    }
+    
+    private boolean isSearchPanelVisible() {
+        return searchPage.isElementPresent(By.id("filterQuery"))
+            && searchPage.isElementPresent(By.id("searchButton"));
```
Comment (JosephineKwa) [inline src/test/java/teammates/test/cases/ui/browsertests/AdminSearchPageUiTest.java]: If you are checking for the visibility of the search panel, you might want to use isElementVisible instead. Or If you are checking for the presence of the search panel, perhaps you could rename the method to `isSearchPanelPresent()` instead

File: src/test/java/teammates/test/cases/ui/browsertests/AdminSearchPageUiTest.java [hunk 2]
```diff
@@ -68,6 +78,94 @@ private AdminSearchPage getAdminSearchPage(String instructorId) {
...
+    private boolean isSearchTableHeaderCorrect(int tableNum) {
+        List<String> expectedSearchTableHeaders;
+        List<String> actualSessionTableHeaders;       
+
+        int numColumns = searchPage.getNumberOfColumnsFromDataTable(tableNum);
+        
+        switch (searchPage.getDataTableId(tableNum)) {
+        // Instructor table
+        case "search_table_instructor":
+            if (numColumns != 5) {
```
Comment (JosephineKwa) [inline src/test/java/teammates/test/cases/ui/browsertests/AdminSearchPageUiTest.java]: You could extract this to a constant in this class. Perhaps something like `INSTRUCTOR_SEARCH_TABLE_NUM_COLUMNS`

File: src/test/java/teammates/test/cases/ui/browsertests/AdminSearchPageUiTest.java [hunk 3]
```diff
@@ -68,6 +78,94 @@ private AdminSearchPage getAdminSearchPage(String instructorId) {
...
+            
+            for (int i = 0 ; i < numColumns ; i++) {
+                actualSessionTableHeaders.add(searchPage.getHeaderValueFromDataTable(tableNum, 0, i));
+            }
+            
+            break;
+            
+        // Student table
+        case "search_table":
+            if (numColumns != 6) {
```
Comment (JosephineKwa) [inline src/test/java/teammates/test/cases/ui/browsertests/AdminSearchPageUiTest.java]: Similarly, extract this magic number as well :)

**Comment by taniach** [comment]
@JosephineKwa ready for review again

**Comment by JosephineKwa** [comment]
Should be good to merge

**Pull request merged**